src/extension/content-script/batteries/Reddit.ts

```diff
@@ -0,0 +1,76 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+const urlMatcher = /^https:\/\/www.reddit\.com\/user\/(\w+).*/;
+
+// can we detect the user description from the description metatag?
+function isOnProfilePage() {
+  return (
+    document.querySelector<HTMLMetaElement>(
+      'head > meta[name="description"]'
+    ) != null
+  );
+}
+
+function getUserData() {
+  if (isOnProfilePage()) {
```

Comment:
I am wondering if we need to check this on reddit. 
On twitter we for example need to somehow differentiate between `/home` `/notifications` and `/your-username`. but on reddit it seems we can do that directly from the URL `reddit.com/user/...` I guess if our regex matches then we can be sure to be on a user profile.